sqlobject/src/main/java/org/jdbi/v3/sqlobject/locator/ClasspathSqlLocator.java

```diff
@@ -0,0 +1,23 @@
+package org.jdbi.v3.sqlobject.locator;
+
+import org.jdbi.v3.core.config.ConfigRegistry;
+import org.jdbi.v3.sqlobject.internal.SqlAnnotations;
+
+import java.lang.reflect.Method;
+import java.util.function.Function;
+
+/**
+ * @see org.jdbi.v3.core.locator.ClasspathSqlLocator
+ */
+public class ClasspathSqlLocator implements SqlLocator {
+    @Override
+    public String locate(Class<?> sqlObjectType, Method method, ConfigRegistry config) {
+        Function<String, String> valueOrMethodNameToSql = key -> {
+            String filename = key.isEmpty() ? method.getName() : key;
+            return org.jdbi.v3.core.locator.ClasspathSqlLocator.findSqlOnClasspath(sqlObjectType, filename);
```

Comment:
I could see this getting confusing.

Reply:
How about calling it `AnnotatedClasspathSqlLocator` or something so as to disambiguate
Also it can probably be non-public

Reply:
Do you mean renaming the one in core or the one in sqlobject?

Reply:
The new one added right here.  Or maybe `SqlObjectClasspathSqlLocator`.  Having two classes with exactly the same name is unfortunate.

Reply:
Yeah I agree, but at the same time it kinda makes sense to have the same name. I'll switch it to `SqlObjectClasspathSqlLocator`.